tests/integrations/server/lock_manager.rs

```diff
@@ -159,13 +159,11 @@ fn must_merge_region(
     cluster.must_put(target_region_key, b"v");
 }
 
-fn find_peer_of_store(region: &Region, store_id: u64) -> Peer {
+fn find_peer(region: &Region, store_id: u64) -> Option<&Peer> {
```

Comment:
Not as clear as before. When seeing a function name `find_peer` that accepts a id, it's usually assumed the id is peer id.